### ISSUE TITLE
Convert to component library

### ISSUE THREAD

**Issue by pi0** [Dashibase/lotion#7]
Hi! I just saw your work and it is very impressive and minimal 💯 

How do you think if we refactor some parts of the core to publish as a component library to npm? It could be a really nice minimal editor component.

**Comment by greentfrapp** [comment]
Yes! We're working on it. The two main parts we're looking at releasing soon:
- Registering a custom component (including documentation)
- Publishing this as an npm library
Some WIP is going on in the add-custom-block branch if you're interested. We should be releasing that in a few days.
